screens/RecipeCard.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
This recipe card is not being used currently. Maybe can use in future but for not we can just remove it I think. Still do need to update though the recipe details screen to allow it to have the save recipe functionality as well.